tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java

```diff
@@ -120,7 +120,7 @@ private File checkStaticServicesFile() throws IOException
                 }
             }
 
-            if( !supervisor.getAgent().redirect( -1 ) )
+            if( !supervisor.getAgent().redirect( Agent.COMMAND_SESSION ) )
```

Comment:
This service command seems to be unrelated to this issue.  If this is a bug can you submit it in another ticket where we discuss the problems in new JIRA?  thanks!